Formula/n98-magerun.rb

```diff
@@ -0,0 +1,17 @@
+class N98Magerun < Formula
+  desc "Swiss army knife for Magento developers, sysadmins and devops."
+  homepage "http://magerun.net/"
+  url "https://files.magerun.net/n98-magerun-1.97.22.phar"
+  version "1.97.22"
+  sha256 "199dd0d416260c68e691580ffd3f7a448d60f616529aadadcfee584992d6253e"
+
+  bottle :unneeded
+
+  def install
+    bin.install "n98-magerun.phar" => "n98-magerun"
```

Comment:
You'll need to adjust this line, so that it takes the versioned file name into account. (The installation currently fails.) You can use the following, so this line won't need to be changed with every update:

```ruby
bin.install "n98-magerun-#{version}.phar" => "n98-magerun"
```